Gaijinizer patch/Map101.txt

```diff
@@ -32,8 +34,9 @@ about waist ...#MTLed
 > CONTEXT: Map101/events/10/pages/0/6/Dialogue/kazuya
 > CONTEXT: Map101/events/12/pages/0/6/Dialogue/kazuya
 > CONTEXT: Map101/events/13/pages/0/6/Dialogue/kazuya
-...... After all? I also thought that I was
-getting too crowded behind the park ...#MTLed
+...Really? I thought it was strange too that
+there'd be so many trees in the back of the
+park...#Ron
```

Comment:
Minor flow:
>_**Really?
I thought it was also strange that there'd be
so many trees behind the park...**_